### PR TITLE
Convert CI to GitHub Actions

### LINKED ISSUE #7: TST: Use GitHub Actions
Please see https://groups.google.com/g/astropy-dev/c/45frnTEAX-U

### PR DESCRIPTION
**Pull request by jonathansick**
This drops Travis CI.

We'll want to modernize some of the testing infrastructure for consistency (e.g. using tox), but this update puts us in a good place going forwards.

Fixes #7

### CODE REVIEW

**Comment by adrn** [comment]
LGTM!

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          - 3.7
+          - 3.8
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history for setuptools_scm
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
```
Comment (pllim) [inline .github/workflows/ci.yaml:22]: Why not `v2`?

**Comment by pllim** [comment]
Thank you!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (jonathansick) [reply .github/workflows/ci.yaml:22]: My bad! I was quickly copying and pasting from another project that I hadn't updated lately. Will fix up later 👍

**Pull request merged**